README.md

```diff
@@ -212,7 +212,7 @@ QuantumKatas/
   BasicGates/
     README.md                  # Instructions specific to this kata.
     .vscode/                   # Metadata used by Visual Studio Code.
-    BasicGates.sln             # Visual Studio 2019 solution file.
+    BasicGates.sln             # Visual Studio 2022 solution file.
```

Comment:
You can remove version here, since technically it's still a VS 2019 project

Reply:
Sounds good!